Code/GraphMol/Descriptors/testBCUT.cpp

```diff
@@ -0,0 +1,47 @@
+//
+//  Copyright (C) 2020 Brian P. Kelley
+//   @@ All Rights Reserved @@
+//
+//  This file is part of the RDKit.
+//  The contents are covered by the terms of the BSD license
+//  which is included in the file license.txt, found at the root
+//  of the RDKit source tree.
+//
+
+#include <RDGeneral/test.h>
+#include <RDGeneral/Invariant.h>
+#include <GraphMol/RDKitBase.h>
+#include <GraphMol/FileParsers/MolSupplier.h>
+#include <GraphMol/FileParsers/FileParsers.h>
+#include <RDGeneral/RDLog.h>
+#include <vector>
+#include <algorithm>
+#include <fstream>
+
+#include <GraphMol/Descriptors/BCUT.h>
+
+void test1(){
```

Comment:
pre-merge this will require some real testing

Reply:
I'm getting the original paper to see if there are any examples, it will take a couple of days.

Reply:
Maybe just include the same molecules that you're testing in Python here?
I'd prefer to have the larger test case here and then in Python you can just include a couple to test that the functions work instead of correctness of the underlying C++.